translate.sh

```diff
@@ -0,0 +1,5 @@
+export GOOGLE_APPLICATION_CREDENTIALS=../key.json
```

Comment:
can you put this file into the tools folder

Reply:
understood. will do.